portfolio/src/main/webapp/script.js

```diff
@@ -202,6 +214,12 @@ function createCommentElement(comment) {
 }
 
 function getNumberOfComments() {
-  const numberOfCommentsMenu = document.getElementById("numberOfComments");
+  const numberOfCommentsMenu = document.getElementById('numberOfComments');
   return numberOfCommentsMenu.options[numberOfCommentsMenu.selectedIndex].value;
 }
+
+function createMap() {
```

Comment:
It probably makes sense to pass the map element to this function from the call site since it is already a variable there.